cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -848,8 +848,8 @@ pub enum EditResult {
     /// The manifest was modified, although the change could be as minimal as
     /// whitespace
     Changed {
-        old_lockfile: Option<Lockfile>,
-        new_lockfile: Lockfile,
+        old_lockfile: Box<Option<Lockfile>>,
+        new_lockfile: Box<Lockfile>,
```

Comment:
question: did the struct get too large or why do we need to start box'ing them here?
I assume these rust changes are a result of also bumping the rust toolchains which brought new clippy lints?

Reply:
Yeah clippy complained, I assume because of a newer clippy